components/Layout/Header.tsx

```diff
@@ -239,84 +257,103 @@ const MenuToggle = styled.button<{ isLight?: boolean }>`
+        ${isLightCoWAMM && `background: url(${MenuImageThemedCoWAMM}) no-repeat center/contain`};
+      `}
   }
 
   ${Media.mediumDown} {
     display: flex;
   }
 `
 
-const Logo = styled.div<{ isLight?: boolean; menuVisible?: boolean }>`
+const Logo = styled.div<{ isLight?: boolean; isLightCoWAMM?: boolean; menuVisible?: boolean; isSticky?: boolean }>`
   width: 12.2rem;
   height: 3.8rem;
-  background: url(${LogoImage}) no-repeat center/contain;
-  ${({ isLight }) => !isLight && `background: url(${LogoLightImage}) no-repeat center/contain`};
+  background: ${({ isLight, isLightCoWAMM }) =>
+      `url(${isLightCoWAMM ? LogoLightImageThemedCoWAMM : !isLight ? LogoLightImage : LogoImage})`}
+    no-repeat center/contain;
   cursor: pointer;
   z-index: 10;
 
-  .sticky & {
-    ${({ isLight }) => !isLight && `background: url(${LogoImage}) no-repeat center/contain`};
-  }
+  ${({ isSticky, isLightCoWAMM }) =>
+    isSticky &&
+    css`
+      background: ${`url(${isLightCoWAMM ? LogoImageThemedCoWAMM : LogoImage})`} no-repeat center/contain;
+    `}
 
   ${Media.mediumDown} {
     background: url(${LogoIconImage}) no-repeat center/contain;
     ${({ isLight, menuVisible }) =>
       (!isLight || menuVisible) && `background: url(${LogoIconLightImage}) no-repeat center/contain`};
+
+    ${({ isLightCoWAMM, menuVisible }) =>
+      (isLightCoWAMM || menuVisible) && `background: url(${LogoIconLightImageThemedCoWAMM}) no-repeat center/contain`};
+
     width: 3.6rem;
     height: 3.2rem;
     background-size: contain;
     background-position: left;
 
-    .sticky & {
-      width: 3.6rem;
-      height: 3.2rem;
-      background: url(${LogoIconImage}) no-repeat center/contain;
-    }
+    ${({ isSticky, isLightCoWAMM, menuVisible }) =>
+      isSticky &&
+      css`
+        width: 3.6rem;
+        height: 3.2rem;
+        background: url(${LogoIconImage}) no-repeat center/contain;
+        ${(isLightCoWAMM || menuVisible) && `background: url(${LogoIconImageThemedCoWAMM}) no-repeat center/contain`};
+      `}
   }
 `
 
 interface Props {
   isLight?: boolean
+  isLightCoWAMM?: boolean
 }
 
-export default function Header({ isLight = false }: Props) {
+export default function Header({ isLight = false, isLightCoWAMM = false }: Props) {
   const swapURL = CONFIG.url.swap
   const isTouch = useMediaQuery(`(max-width: ${Media.mediumEnd})`)
   const [menuVisible, setIsMenuVisible] = useState(false)
   const toggleBodyScroll = () => {
     !menuVisible ? document.body.classList.add('noScroll') : document.body.classList.remove('noScroll')
   }
-  const handleClick = () => {
+  const handleClick = ({ label }) => {
+    // log GA event
+    sendGAEventHandler({ category: GAEventCategories.NAVIGATION, action: `Header menu clicked: ${label}` })
+
     if (isTouch) {
       setIsMenuVisible(!menuVisible)
       toggleBodyScroll()
     }
   }
 
   return (
-    <InView threshold={1} delay={500}>
+    <InView threshold={1} delay={500} initialInView={true}>
```

Comment:
I think you don't need to include the `{true} ` 

I think this should also work:

<InView threshold={1} delay={500} initialInView>